person.rb

```diff
@@ -1,8 +1,8 @@
 require './nameable'
-
+require './book'
 # Class Person
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals, :parent_permession
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permession: true)
```

Comment:
- [ ] Are you sure it's `parent_permession` or is it `parent_permission`? 😊 Kindly fix the typo 📝